docs/man_pages/project/creation/sample-hybrid.md

```diff
@@ -0,0 +1,31 @@
+sample hybrid
+==========
+
+Usage | Syntax
+------|-------
+General | `$ appbuilder sample hybrid`    
+
+Lists all available Cordova sample apps with name, description, GitHub repository, and clone command. To clone a selected sample app,
```

Comment:
Lists all available Apache Cordova sample apps with name, description, GitHub repository and clone command. To clone a selected sample app, run its clone command as it is listed.